scripts/build-rpm.sh

```diff
@@ -32,6 +32,13 @@ description='drush: Pantheon rpm containing commandline tool for Drupal'
 versions=$(cat $bin/../VERSIONS.txt | grep -v '^#')
 
 for version in $versions; do(
+
+	if [[ $version == "6-2-remove-uname" ]] || [[ $version == "5-10-production" ]]
+	then
+		version_untampered=$version
+		version=$(echo $version | sed -e 's/\([^-]*\)-\([^-]*\)-\(.*\)/\1.\2.0\-\3/')
```

Comment:
Confirm that version is unchanged when version is a SemVer.  Should not be altered.